packages/x-charts/src/Gauge/gaugeClasses.ts

```diff
@@ -24,5 +24,3 @@ export const gaugeClasses: GaugeClasses = generateUtilityClasses('MuiGauge', [
   'referenceArc',
   'valueText',
 ]);
-
-export default gaugeClasses;
```

Comment:
Can't this break user imports? E.g., if they're doing `import gaugeClasses from '@mui/x-charts/Gauge/gaugeClasses`?

I'm ok with removing it, but I'm wondering if we want to add notice of this breaking change?

Reply:
AFAIK we only support breaking changes up to the third level `import gaugeClasses from '@mui/x-charts/Gauge`

Reply:
Didn't know that, thanks 😄